services/web-server/src/main.js

```diff
@@ -150,7 +150,7 @@ const load = loader(
           validationRules: [
             queryLimit(1000),
             depthLimit(10),
-            createComplexityLimitRule(1000),
+            createComplexityLimitRule(4500),
```

Comment:
lol

Reply:
🙈 .. hooks query is quite aggressive.. fetching all groups+hooks, and then fetching last fire details, which combined produces a very complex query.. but relatively fast despite